Sources/JsonModel/ResultData/FileArchivable.swift

```diff
@@ -0,0 +1,453 @@
+        case "iPhone12,3":                                  return "iPhone 11 Pro"
+        case "iPhone12,5":                                  return "iPhone 11 Pro Max"
+        case "iPhone12,8":                                  return "iPhone SE (2nd generation)"
+        case "iPhone13,1":                                  return "iPhone 12 mini"
+        case "iPhone13,2":                                  return "iPhone 12"
+        case "iPhone13,3":                                  return "iPhone 12 Pro"
+        case "iPhone13,4":                                  return "iPhone 12 Pro Max"
+        case "iPhone14,4":                                  return "iPhone 13 mini"
+        case "iPhone14,5":                                  return "iPhone 13"
+        case "iPhone14,2":                                  return "iPhone 13 Pro"
+        case "iPhone14,3":                                  return "iPhone 13 Pro Max"
+            
+        case "iPad2,1", "iPad2,2", "iPad2,3", "iPad2,4":    return "iPad 2"
+        case "iPad3,1", "iPad3,2", "iPad3,3":               return "iPad 3"
+        case "iPad3,4", "iPad3,5", "iPad3,6":               return "iPad 4"
+        case "iPad4,1", "iPad4,2", "iPad4,3":               return "iPad Air"
+        case "iPad5,3", "iPad5,4":                          return "iPad Air 2"
+        case "iPad6,11", "iPad6,12":                        return "iPad 5"
+        case "iPad2,5", "iPad2,6", "iPad2,7":               return "iPad Mini"
+        case "iPad4,4", "iPad4,5", "iPad4,6":               return "iPad Mini 2"
+        case "iPad4,7", "iPad4,8", "iPad4,9":               return "iPad Mini 3"
+        case "iPad5,1", "iPad5,2":                          return "iPad Mini 4"
+        case "iPad6,3", "iPad6,4":                          return "iPad Pro 9.7 Inch"
+        case "iPad6,7", "iPad6,8":                          return "iPad Pro 12.9 Inch"
+        case "iPad7,1", "iPad7,2":                          return "iPad Pro 12.9 Inch 2. Generation"
+        case "iPad7,3", "iPad7,4":                          return "iPad Pro 10.5 Inch"
+        case "iPad7,5", "iPad7,6":                          return "iPad 6"
+        case "iPad7,11","iPad7,12":                         return "iPad 7 10.2 Inch"
+        case "iPad8,1","iPad8,2","iPad8,3","iPad8,4":       return "iPad Pro 3 11 Inch"
+        case "iPad8,5","iPad8,6","iPad8,7","iPad8,8":       return "iPad Pro 3 12.9 Inch"
+        case "iPad11,1","iPad11,2":                         return "iPad Mini 5"
+        case "iPad11,3","iPad11,4":                         return "iPad Air 3"
+            
+        case "AppleTV5,3":                                  return "Apple TV"
+        case "AppleTV6,2":                                  return "Apple TV 4K"
+            
+        case "AudioAccessory1,1":                           return "HomePod"
+            
+        case "i386", "x86_64":                              return "Simulator"
```

Comment:
At some point one of us or SDP will get an Apple Silicon Mac and we'll need to update this for those. Just something to keep in the back of your mind.